download/tasks.py

```diff
@@ -116,6 +117,14 @@ def download_file(ctx, filename, url, max_age=None, md5_url=None):
     save_file_status(ctx, filename, {"last_update": datetime.utcnow(), "md5": md5})
 
 
+@task
+def file_exists(ctx, path):
+    """
+    Print '1' if the provided path exists and is a file, overwise, print '0'.
+    """
+    print(int(os.path.isfile(path)))
```

Comment:
Maybe it would be more standard to use different exit codes for such a task ?
pyinvoke provides an `Exit` exception that could be useful.
```suggestion
@task
def file_exists(ctx, path):
   if not os.path.isfile(path):
       raise invoke.Exit(code=1)
```

And I think `docker-compose run` should forward this exit code.

Reply:
It seems quite cleaner, I was afraid of how `ctx.run` from invoke would handle error codes, turns out that it won't raise an exception with parameter `warn=True`.